### DISCUSSION TITLE
I can't find any log.

### DISCUSSION THREAD

**Discussion by Foreist** [ggml-org/llama.cpp#9634] [Q&A]
I was setting below 
`CUDA_VISIBLE_DEVICES="3" LLAMA_ARG_N_GPU_LAYERS=100 /workspace/llama.cpp/llama-server --verbose --verbose-prompt --log-enable --log-file '/workspace/model_serving/llamacpp' -np 17 -m '/workspace/llama.cpp/models/llama3.1-8B-Instruct.Q4_K_M.gguf' --host '0.0.0.0' --port 20022 -t 12 -b 256 -c 20000 -fa`
llamacpp created "/workspace/model_serving/llama.log", "/workspace/model_serving/llamacpp.log"
but I found out any context in "workspace/model_serving/llamacpp.log"
[1727229882] warming up the model with an empty run

Please tell me how to log.

**Comment by ngxson** [comment]
Simply adding `--log-file` works for me: `./llama-server -m ... --log-file test.log`
